go/vt/vtctl/reparentutil/util.go

```diff
@@ -151,17 +153,17 @@ func ElectNewPrimary(
 	}
 
 	// sort the tablets for finding the best primary
-	err = sortTabletsForReparent(validTablets, tabletPositions, innodbBufferPool, opts.durability)
+	err = sortTabletsForReparent(validTablets, tabletPositions, innodbBufferPool, backingUpTablets, opts.durability)
 	if err != nil {
 		return nil, err
 	}
 
 	return validTablets[0].Alias, nil
 }
 
-// findPositionAndLagForTablet processes the replication position and lag for a single tablet and
+// findPositionLagBackingUpForTablet processes the replication position and lag for a single tablet and
 // returns it. It is safe to call from multiple goroutines.
-func findPositionAndLagForTablet(ctx context.Context, tablet *topodatapb.Tablet, logger logutil.Logger, tmc tmclient.TabletManagerClient, waitTimeout time.Duration) (replication.Position, time.Duration, error) {
+func findPositionLagBackingUpForTablet(ctx context.Context, tablet *topodatapb.Tablet, logger logutil.Logger, tmc tmclient.TabletManagerClient, waitTimeout time.Duration) (replication.Position, time.Duration, bool, error) {
```

Comment:
```suggestion
func findPositionLagAndBackupStatusForTablet(ctx context.Context, tablet *topodatapb.Tablet, logger logutil.Logger, tmc tmclient.TabletManagerClient, waitTimeout time.Duration) (replication.Position, time.Duration, bool, error) {
```

Reply:
Maybe we can come up with some shorter function name for this.

Reply:
I settled for `findTabletPositionLagBackupStatus()`, I hope it's acceptable